cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkProgramRuntimeProvider.java

```diff
@@ -68,9 +69,11 @@ public ProgramRunner createProgramRunner(ProgramType type, Mode mode, Injector i
    */
   private ProgramRunner createSparkProgramRunner(Injector injector, String programRunnerClassName) {
     try {
-      CConfiguration cConf = injector.getInstance(CConfiguration.class);
-      final SparkRunnerClassLoader classLoader = new SparkRunnerClassLoader(getClassLoaderURLs(cConf),
-                                                                            getClass().getClassLoader());
+      boolean rewriteYarnClient = injector.getInstance(CConfiguration.class)
```

Comment:
This is the current description: 
Specify whether to rewrite the yarn.Client class in Spark to work around the issue SPARK-13441 in CDM clusters.

Do we need to update it?